columnflow/tasks/reduction.py

```diff
@@ -113,7 +113,15 @@ def run(self):
         write_columns: set[Route] = set()
         skip_columns: set[str] = set()
         for c in self.config_inst.x.keep_columns.get(self.task_family, ["*"]):
-            for r in (self.find_keep_columns(c) if isinstance(c, ColumnCollection) else {Route(c)}):
+            # expand collections and braces in strings
+            if isinstance(c, ColumnCollection):
+                routes = self.find_keep_columns(c)
+            elif isinstance(c, str):
+                routes = set(map(Route, law.util.brace_expand(c)))
+            else:
+                routes = {Route(c)}
```

Comment:
This code block of determining the routes seems to repeat quite often. Does it make sense to refactor this?

Reply:
Yup, did that in a new method :+1: